static/css/CardComponent.scss

```diff
@@ -0,0 +1,87 @@
+@mixin aspect-ratio($width, $height) {
```

Comment:
making a ticket and assigning myself, feel free to assign yourself as well. Want to use the scss file structure I've found beneficial to organize stylesheets on this proj and allow them to be compiled to one minified file. Fine with this for the time being though